content/Setting Up Firebase on Android- Amaury Ricardo.md

```diff
@@ -0,0 +1,113 @@
+
+### What can we do with Firebase?
+
+Firebase provides a lot of tools that we will talk about in next posts but resuming Firebase allow as to:
+- Push notifications
+- Analytics
+- A/B Testing
+- Authentication
+- Real time db
+- App distributions
+- Google cloud: Hosting, Storage
+- Machine Learning
+
+and a lot of others tools that I invited you to test.
+
+### Setting up on Android
+
+1. **Adding dependencies**
+
+On your project  Gradle file "build.gradle"
+
+    ```java
+    repositories {
+        google()
+        jcenter()
+        maven {
+            url 'https://maven.fabric.io/public'
+        }
+    }
+
+    dependencies {
+    		....        
+    		classpath 'com.google.gms:google-services:4.3.2'  // Google Services plugin
+        classpath 'io.fabric.tools:gradle:1.31.2'  // Crashlytics plugin
+    		....
+    }
+
+    ```
+
+On your module (app) Gradle file "build.gradle"
+
+    ```java
+    apply plugin: 'com.android.application'
+
+    android {
+       ....
+    }
+    dependencies {
+    		.....
+    	// Add the Firebase SDK for Google Analytics
+      implementation 'com.google.firebase:firebase-analytics:17.2.1'
+    }
+
+    // Add the following line to the bottom of the file
+    apply plugin: 'com.google.gms.google-services'
+    ```
+
+2. **Create the project on your Firebase console**
+
+    Open your [firebase console](https://console.firebase.google.com/)  and add a new project:
+
+    ![Add firebase](images/fb1.png)
+
+    Once you create the project in order to add a new Android app:
+
+    ![Add android plattform](images/fb2.png)
+
+    On the next steps enter the Android applicationId , you can found on the Gradle file:
+
+    ```kotlin
+    ....
+    android {
+        ....
+        defaultConfig {
+            applicationId "com.rootstrap.android"
+    		....
+    ```
+
+    **Note: In case you configure several build types with different package names you need to add an Android app to the current Firebase console for each of them.**
+
+    ```kotlin
+    productFlavors {
+            development {
+                dimension "server"
+    				//Fore this case you have to create a new Android app 
+    				// in to your current Firabese console
+    				// applicationId = com.rootstrap.android.dev
+                applicationIdSuffix ".dev"
+            }
+
+            production {
+                dimension "server"
+            }
+    }
+    ```
+
+Once you create the app, download the google-services.json file and copy on your module folder (app).
+
+3. **Finally run your App, you should be able to see on your Project overview 1 daily active user (your current device or emulator)**
```

Comment:
use lowercased app since it is not the actual name of the app and to keep consistency with the rest of the post.

Reply:
The second part of of the sentece(after the comma) should be below the title since it is the output of the action. This way you have a shorter and clearer action 'Finally, run your app'.